docs/UserGuide.md

```diff
@@ -181,31 +278,57 @@ Format: `friend --schedule FRIEND_ID -p START_HOUR END_HOUR DAY -f IS_FREE`
 
 **:information_source: Notes about the schedule:**<br>
 
-New friends start off which a schedule with all periods marked as busy.
+* New friends start off which a schedule with all periods marked as busy.
+
+* gitGud displays the periods of time friends are free using the 24-hour notation.
+The end of a day, midnight (12am) is displayed as `2400` to allow users to easily differentiate it 
+from the start of the day, which is displayed as `0000`.
 
 </div>  
 
+Command constraints: 
 * `FRIEND_ID` must belong to an existing friend in gitGud.
 * `START_HOUR` and `END_HOUR` must be an integer between 0 and 24 inclusive, and represents the hour of the day.
   * `END_HOUR` must be strictly after `START_HOUR`.
   * For example, `START_HOUR` = 2 and `END_HOUR` = 20 represents the time period from the 2nd to the 20th hour of the day.
 * `DAY` must be an integer between 1 and 7 inclusive, with each day representing a day of the week from Monday to Sunday.
-* `IS_FREE` is used to mark the period as a free or busy period, with `1` meaning free and `0` meaning busy.
-* gitGud's UI displays the periods of time friends are free using the 24-hour notation.
-  * The end of a day, midnight (12am) is displayed as 2400 to allow users to easily differentiate it from the start of the day, which is displayed as 0000.
+* `IS_FREE` is used to mark the period as a free or busy period, and can only be the values `1` meaning free or `0` meaning busy.
 
 Examples:
 * `friend --schedule Draco -p 18 22 2 -f 1` Schedules "Draco" as free from 18th to 22nd hour, 1800 - 2200 on Tuesday.
 * `friend --schedule Draco -p 12 24 7 -f 0` Schedules "Draco" as busy from 12th to 24th hour, 1200 - 2400 (midnight) on Sunday.
 
-Snapshot:
+Screenshot:
+
+![Ui](images/ui-screenshots/friend-schedule.png)
+<br><center><ins>Image: gitGud after scheduling a friend's availability</ins></center>
+
+#### Recommending friends available to play with - `recommend`
+
+Lists all friends who both play the given game with `GAME_ID` and are available to play
+with at the specified time each week based on their stored schedules.
+
+Format: `recommend -g GAME_ID -t HOUR DAY`
+
+Command constraints:
+* The `HOUR` provided must be an integer within 0 to 23 inclusive, with 0 representing midnight `0000` of the current
+  day and 23 representing `2300`.
```

Comment:
Could potentially be confusing when saying 0 represents midnight of the current day. Perhaps we can standardise with schedule's phrasing which is to call 0000 the start of the day.